include/swift/SIL/SILLinkage.h

```diff
@@ -86,10 +86,15 @@ enum {
   NumSILLinkageBits = 3
 };
 
-/// Related to linkage: flag if a function or global variable is fragile.
-enum IsFragile_t {
-  IsNotFragile,
-  IsFragile
+/// Related to linkage: flag if a function or global variable is serialized,
+/// either unconditionally, or if referenced from another serialized function.
+enum IsSerialized_t : unsigned char {
+  // Never serialized.
+  IsNotSerialized,
+  // Serialized if referenced from another serialized function.
+  IsSerializable,
+  // Always serialized.
+  IsSerialized
```

Comment:
IMHO, `IsSerialized` is not a very good choice for a name. It is too easy to think that something was serialized by the SIL serializer already. I'd prefer names which would use a different verb. Or they could indicate the possibility, e.g. `NotToBeSerialized`, `CanBeSerialized`, `ShouldBeSerialized`....